src/main/kotlin/no/nav/klage/oppgave/service/BehandlingService.kt

```diff
@@ -111,6 +113,22 @@ class BehandlingService(
             }
         }
 
+        if (behandling.mottattKlageinstans == null) {
+            behandlingValidationErrors.add(
+                InvalidProperty(
+                    field = "mottattKlageinstans",
+                    reason = "Sett en dato på for Anke mottatt."
```

Comment:
Vet vi det er en anke? Skal det være fordi mottakKlageinstans skal være nullable for anker?